### PR TITLE
Docs-Rastreabilidade

### PR DESCRIPTION
**Pull request by IanLucca12**
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to <repo_name>?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the CONTRIBUTING.md doc.
- [x] I have added necessary documentation (if appropriate).
- [x] Any dependent changes have been merged and published in downstream modules.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/Artefatos/Ishikawa.md [hunk 1]
```diff
@@ -5,38 +5,38 @@ O Diagrama de Causa-Efeito, também conhecido como Diagrama de Ishikawa ou Diagr
...
 
 **Demanda restrita**: A procura por essas plataformas é limitada, talvez por serem nichos específicos ou pela falta de conscientização sobre os benefícios dessas soluções.
 
 ## Funcionalidade:
-<p align="justify">&emsp;&emsp;Envolve os recursos e capacidades específicas que a plataforma oferece. Para ser eficaz, a plataforma deve incluir uma variedade de funcionalidades que atendam às necessidades dos usuários, como a capacidade de criar e gerenciar campeonatos em tempo real.
+<p align="justify">&emsp;&emsp;Envolve os recursos e capacidades específicas que a plataforma oferece. Para ser eficaz, a plataforma deve incluir uma variedade de <a href="https://unbarqdsw2024-2.github.io/2024.2_G4_Esporte_Entrega_01/#/Artefatos/Glossario">bugs</a> que atendam às necessidades dos <a href="https://unbarqdsw2024-2.github.io/2024.2_G4_Esporte_Entrega_01/#/Artefatos/Glossario">funcionalidades</a>, como a capacidade de criar e gerenciar campeonatos em tempo real.
```
Comment (FabricioDeQueiroz) [inline docs/Artefatos/Ishikawa.md:20]: Mudar o texto dentro da tag <a>.

Comment (FabricioDeQueiroz) [reply docs/Artefatos/Ishikawa.md:20]: Se possível usar o formato `[Texto](Link)` também, para manter padronizado.

**Review by FabricioDeQueiroz — CHANGES_REQUESTED** [review]
Pequenas correções

**Review by harry-cmartin — APPROVED** [review]
LGTM

**Review by gustavofbs — APPROVED** [review]
LGTM

**Review by FabricioDeQueiroz — APPROVED** [review]

**Pull request merged**